cloud/ovh/ovh_ip_loadbalancing_backend.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import sys
+try:
+    import ovh
+    import ovh.exceptions
+    HAS_OVH = True
+except ImportError:
+    HAS_OVH = False
+
+# import module snippets
+from ansible.module_utils.basic import *
+# and APIError from ovh api
+from ovh.exceptions import APIError
```

Comment:
this one should be in the try/except clause, since it will fail with a traceback if ovh module is not installed